mondrian/src/main/java/mondrian/rolap/SqlContextConstraint.java

```diff
@@ -282,10 +282,8 @@ public void addMemberConstraint(
             evaluator.restore(savepoint);
         }
 
-        // comment out addMemberConstraint here since constraint
-        // is already added by addContextConstraint
-        // SqlConstraintUtils.addMemberConstraint(
-        //        sqlQuery, baseCube, aggStar, parent, true);
+         SqlConstraintUtils.addMemberConstraint(
```

Comment:
@mkambol well, at least I didn't found the addition of the "WHERE parent = value" constraint  in SqlConstraintUtils.addContextConstraint